test/new-e2e/tests/installer/host/fixtures.go

```diff
@@ -94,3 +101,69 @@ func (h *Host) SetBrokenDockerConfigAdditionalFields() {
 func (h *Host) RemoveBrokenDockerConfig() {
 	h.remote.MustExecute("sudo rm /etc/docker/daemon.json")
 }
+
+// SetupFakeAgentXP sets up a fake Agent XP with configurable options.
+func (h *Host) SetupFakeAgentXP() FakeAgent {
```

Comment:
Nit: Let's keep consistent with the `Exp` suffix or align on `XP`, on first read it was impossible to understand what was going on